website/docs/learning-and-resources/how-to-guides/how-to-work-with-local-apis-on-appsmith.mdx

```diff
@@ -0,0 +1,119 @@
+---
+
+sidebar_position: 5
+
+description: >-
+  This guide helps you to connect your APIs or databases running on localhost
+  with Appsmith using host.docker.internal or ngrok.
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Connect to Localhost Databases and APIs
+
+This page describes how to connect a database or API that is hosted locally on the same machine as your Appsmith instance.
+
+
+<Tabs>
```

Comment:
Let's probably avoid tabs for this and keep them as separate sections, as these appear to be different scenarios altogether.